java/memory/src/main/java/io/netty/buffer/NettyArrowBuf.java

```diff
@@ -404,7 +407,7 @@ protected int _getUnsignedMediumLE(int index) {
     this.chk(index, 3);
     long addr = this.addr(index);
     return PlatformDependent.getByte(addr) & 255 |
-            (Short.reverseBytes(PlatformDependent.getShort(addr + 1L)) & '\uffff') << 8;
+        (Short.reverseBytes(PlatformDependent.getShort(addr + 1L)) & '\uffff') << 8;
```

Comment:
(Maybe it is irrelavant to this PR). Here a single call to PlatformDependent.getShort should be sufficient?
It seems reverseBytes and then shift left amounts to directly getting the lower bits?

Reply:
I have added https://issues.apache.org/jira/browse/ARROW-9148 to address this

Reply:
Thanks for your effort.